cobbler/actions/mkloaders.py

```diff
@@ -26,6 +25,8 @@ def __init__(self, api):
         """
         self.logger = logging.getLogger()
         self.bootloaders_dir = pathlib.Path(api.settings().bootloaders_dir)
+        self.bootloc = api.settings().tftpboot_location
+        self.grubconfig_dir = api.settings().grubconfig_dir
```

Comment:
Those are paths, so I would love if we could use them also as Path objects. This also ensures then when being cast to a str that they should have a slash either always appended or never. This should prevent things like: `/my/bootloc/path//`